### ISSUE TITLE
Multi-Tab Details Panels Cannot be Fully Navigated using Keyboards

### ISSUE THREAD

**Issue by MRichards99** [ral-facilities/datagateway#764]
**Description**:
When on table views which contain multiple tabs in the details panels, the second tab onwards cannot be accessed using keyboard navigation only. You can focus on the first tab (which is being displayed, investigation details in the screenshot), but the next tab press changes focus to the DOI (or a link relating to DOI which I can't spot with my generic data) rather than giving focus to the investigation users tab.:
![image](https://user-images.githubusercontent.com/32678030/126640309-8d64c81c-3021-4e6b-99f2-771e37beb9aa.png)

I've also noticed this in the ISIS instrument view - I think it happens wherever there's a multi-tab details panel.

Opened this issue as a result of #738 

**Acceptance criteria**:
- [ ] A user should be able to select the different tabs on the details panels using only their keyboard

**Comment by MRichards99** [comment]
For clarity, you can only focus on the currently selected tab - so if I clicked on the publications tab using my mouse, then went back to keyboard-only navigation, I can only focus on the publications tab.

**Comment by MRichards99** [comment]
This issue also occurs in DataGateway Download, on the cart and downloads tabs:
![image](https://user-images.githubusercontent.com/32678030/126776753-290d5a1a-0e59-479a-bcf7-81f27bff5bb2.png)

It also occurs on the admin download table, on the maintaince and admin download tabs:
![image](https://user-images.githubusercontent.com/32678030/126776809-8efab820-86cc-4117-a6b5-744b2ce55f23.png)

I think it's a fair assumption that this issue will occur whether these types of tabs are present in DataGateway.

**Comment by MRichards99** [comment]
I've just had a quick look at the code and I think doing a match whole word search for `<Tab` will return all the places these types of tabs are being used?
![image](https://user-images.githubusercontent.com/32678030/126777274-ec9a7921-d9e5-4d14-be46-cdf9bfc0979d.png)

**Comment by MRichards99** [comment]
This issue also occurs on DataGateway Search, on the three entity tabs:
![image](https://user-images.githubusercontent.com/32678030/126778452-90422b31-5bf6-497f-b391-ae84e631efcd.png)

**Comment by GoelBiju** [comment]
@MRichards99 I have just been looking into this on `dataview`, `download` and `search`. Although we can't use the tab button to access the other tabs, the right and left keys allow you to select the tab you want and then by pressing enter you can view the contents of the tab. I had a look at the [examples](https://material-ui.com/components/tabs/) by Material-UI and it seems it is the same behaviour.

**Comment by MRichards99** [comment]
I didn't realise you could use arrow keys on those tabs, I must've missed that! Arrow keys are a standard form of keyboard navigation so I agree this isn't really an issue. I'll close this issue.
